### PR TITLE
feat(styles): hash the style file for versioning

### PR DESCRIPTION
**Pull request by jefflembeck**
Filling in VER_CSS isn't a good experience and it leads to
weird issues with commits

Also:

- nginx addition for a terms page in case we have one
- a bunch of whitespace fixes and syntax fixes to make some
stuff easier to read
- a div removal from the account page that was throwing off things

### CODE REVIEW

**Review by deathbearbrown — APPROVED** [review]

File: www/testcancelled.inc [hunk 1]
```diff
@@ -8,7 +8,7 @@
             $page_description = "Website speed test cancelled.";
             include ('head.inc');
         ?>
-        <link rel="stylesheet" href="<?= "/css/button.css?v={$VER_CSS}" ?>" />
+        <link rel="stylesheet" href="<?= "/css/button.css?v={$VER_BUTTON_CSS}" ?>" />
```
Comment (tkadlec) [inline www/testcancelled.inc:11]: @jefflembeck Does this need to be wrapped in `constant()`?

**Comment by deathbearbrown** [comment]
ship it!

File: www/testcancelled.inc (hunk 1 above)
Comment (deathbearbrown) [reply www/testcancelled.inc:11]: omg I missed that entirely

Comment (jefflembeck) [reply www/testcancelled.inc:11]:
```suggestion
        <link rel="stylesheet" href="/css/button.css?v=<?= constant('VER_BUTTON_CSS') ?>" />
```

**Pull request merged**